chcemvediet/apps/anonymization/cron.py

```diff
@@ -215,6 +218,48 @@ def recognize_attachment():
     else:
         recognize_using_ocr(attachment_normalization)
 
+def anonymize_attachment():
+    attachment_recognition = (AttachmentRecognition.objects
+            .filter(successful=True,
+                    content_type=content_types.ODT_CONTENT_TYPE,
+                    attachment__attachmentanonymization__isnull=True)
+            .first())
+    try:
+        zipfile_attachment = zipfile.ZipFile(StringIO.StringIO(attachment_recognition.content))
+        inforequest = attachment_recognition.attachment.generic_object.branch.inforequest
+        namespace = {u'text': u'urn:oasis:names:tc:opendocument:xmlns:text:1.0'}
+        filename_relative = os.path.join(u'attachment_anonymizations', random_string(10))
+        filename = os.path.join(settings.PROJECT_PATH, u'media/' + filename_relative)
+        with zipfile.ZipFile(filename, u'w') as zf:
```

Comment:
@martinmacko47 Tento subor sluzy ako `file` ku `AttachmentAnonymization`, nie je to pomocny subor.

Reply:
:ok: